src/app.rs

```diff
@@ -59,11 +63,14 @@ pub struct Options {
     pub ethereum: ethereum::Options,
 
     #[clap(flatten)]
-    pub contracts: contracts::Options,
+    pub identity_manager: contracts::Options,
```

Comment:
I'm not sure about this rename. From the perspective of a deployer, this deals with the contract. Altogether, "identity manager" is a nebulous term that means much less than "contract" but what it actually means to mean is "contract".

Reply:
Also it was rolled back by Greg because it broke existing deployments